src/resolver/findExportedReactCreateClassCall.js

```diff
@@ -47,6 +48,17 @@ function findExportedReactCreateClass(
     visitDoWhileStatement: ignore,
     visitForStatement: ignore,
     visitForInStatement: ignore,
+    visitExportDeclaration: function(path) {
+      path = resolveToCallExpression(path);
```

Comment:
Mmh, do we have to introduce `resolveToCallExpression`? My approach would have been to just extract the path from `ExportDeclaration` that we are interested in, and run `resolveToValue` on that.